Source/WebKit/WebProcess/InjectedBundle/API/glib/WebKitWebPage.h.in

```diff
@@ -93,7 +93,7 @@ webkit_web_page_get_id                      (WebKitWebPage *web_page);
 WEBKIT_API const gchar *
 webkit_web_page_get_uri                     (WebKitWebPage *web_page);
 
-WEBKIT_API WebKitFrame *
+WEBKIT_DEPRECATED WebKitFrame *
```

Comment:
This is not enough. You also need to add Deprecated tag to the api docs, in `WebKitWebPage.cpp` add:

```
* Deprecated: 2.46
```

After the Since tag.

You also need to add ignore deprecation guards (G_GNUC_BEGIN_IGNORE_DEPRECATIONS/G_GNUC_END_IGNORE_DEPRECATIONS) in the api tests using the function. I think EWS isn't failing because this is only used in tests running already deprecated APIs that are only built for GTK3 and EWS builds with GTK4.